cmd/run.go

```diff
@@ -610,6 +610,15 @@ func setupExecutors(
 		}
 	}
 
+	if s.WillRunTaskOfType(pipeline.AssetTypeBigquerySeed) {
+		jinjaVariables := jinja.PythonEnvVariables(&startDate, &endDate, pipelineName, runID, fullRefresh)
+		if usePipForPython {
+			mainExecutors[pipeline.AssetTypeBigquerySeed][scheduler.TaskInstanceTypeMain] = python.NewLocalOperator(config, jinjaVariables)
```

Comment:
we should not be using the python operators for the seed, we need a new type of operator